### ISSUE TITLE
Crash during final file sync when file saved at end?

### ISSUE THREAD

**Issue by shawnlewis** [wandb/client#341]
david [12:09 PM]
replied to a thread:
hey! we have a checkpoint of 104.9MB in our wandb folder, and I notice it's not getting uploaded to the sync'd files. Anything I need to do to make sure large checkpoints get sync'd correctly?

shawn [12:14 PM]
It needs to go in the run subdirectory
e.g. `run-20190519_210753-p0uj0rqx`
A
david [12:18 PM]
It is under there. Seems like it actually attempts to sync the checkpoints all at the very end (note the models/iter_000150.model):
```wandb: Syncing files in ../../checkpoints/references/stylegan/wandb/run-20190521_191534-1s9ymt1c:
wandb:   diff.patch
wandb:   models/iter_000150.model
wandb: plus 6 W&B file(s) and 2 media file(s)
wandb:                                                                                
wandb: Synced mnist_stylegan_baseline: https://app.wandb.ai/angle-tech/lab/runs/1s9ymt1c
Exception ignored in: <bound method tqdm.__del__ of Size: 16; G: 28.247; D: 14.896; Grad: 1.000; Alpha: 1.00000:   0% 277/3000000 [02:03<249:42:34,  3.34it/s]>
Traceback (most recent call last):
  File "/home/david/anaconda3/envs/angle/lib/python3.6/site-packages/tqdm/_tqdm.py", line 954, in __del__
  File "/home/david/anaconda3/envs/angle/lib/python3.6/site-packages/tqdm/_tqdm.py", line 1156, in close
  File "/home/david/anaconda3/envs/angle/lib/python3.6/site-packages/tqdm/_tqdm.py", line 511, in _decr_instances
  File "/home/david/anaconda3/envs/angle/lib/python3.6/site-packages/tqdm/_monitor.py", line 51, in exit
  File "/home/david/anaconda3/envs/angle/lib/python3.6/threading.py", line 521, in set
  File "/home/david/anaconda3/envs/angle/lib/python3.6/threading.py", line 364, in notify_all
  File "/home/david/anaconda3/envs/angle/lib/python3.6/threading.py", line 347, in notify
TypeError: 'NoneType' object is not callable

Process finished with exit code 1```

**Comment by vanpelt** [comment]
In this run their model file was actually uploaded.  I think the stacktrace is specific to their code, but I'm not positive.

**Comment by vanpelt** [comment]
Should be fixed in 0.8.2
